### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by SNMellott**
Working on home page stuff and getting that all set up. Switched out some haml files and what have you.

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -1,2 +1,13 @@
 module ApplicationHelper
+
+  #Returns the full title on a per page basis
+  def full_title(page_title = '')
+    base_title = "Organizer Application"
+    if page_title.empty?
```
Comment (ryanzbone) [inline app/helpers/application_helper.rb:6]: maybe `.blank?` would be a better choice here instead?

File: app/views/static_pages/home.html.erb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<div class="jumbotron jumbotron-fluid">
```
Comment (ryanzbone) [inline app/views/static_pages/home.html.erb:1]: Check this out to convert your erbs to hamls 

https://github.com/haml/haml-rails#converting-rails-application-layout-file-to-haml-format

Comment (SNMellott) [reply app/views/static_pages/home.html.erb:1]: Thank you! Haml has been giving me some difficulty. I was trying to right it as hail to start with but I'm not comfortable enough with it yet and it was taking too much time for me to figure out. So I was writing it all out in erb first then switch it to Haml when I finish. :)

File: app/helpers/application_helper.rb (hunk 1 above)
Comment (SNMellott) [reply app/helpers/application_helper.rb:6]: Oh thank you! I'll change it over. :)

**Pull request merged**